edb/pgsql/compiler/__init__.py

```diff
@@ -65,7 +65,7 @@ def compile_ir_to_sql_tree(
         ]
     ] = None,
     backend_runtime_params: Optional[pgparams.BackendRuntimeParams]=None,
-) -> Tuple[pgast.Base, context.Environment]:
```

Comment:
Maybe these functions ought to return a namedtuple or a dataclass. Doesn't need to happen now though.